node_modules/helpers/modules/time/index.js

```diff
@@ -1,25 +1,69 @@
 function getTime() {
     const currentDate = new Date();
 
-    const hour = currentDate.getHours();
+    const hour_24 = currentDate.getHours();     //current hour in 24 hour format
+    const hour_12 = formatHour(hour_24);        //current hour in 12 hour format + AM/PM suffix
     const minutes = currentDate.getMinutes();
     const seconds = currentDate.getSeconds();
-    const time = `${hour}:${minutes}:${seconds}`;
+
+    const time = {
+        hour_24: hour_24,
+        hour_12: hour_12,
+        minutes: minutes,
+        seconds: seconds
+    };
 
     return time;
 }
 
 function getDate() {
     const currentDate = new Date();
 
-    const day = currentDate.getDate();
-    const month = currentDate.getMonth();
+    const day = currentDate.getDay();
+    const dayName = getDayName(day);
+
+    const dayOfMonth = currentDate.getDate();
+
+    const monthNumber = currentDate.getMonth();
+    const monthName = getMonthName(monthNumber);
+
     const year = currentDate.getFullYear();
-    const date = `${day}-${month + 1}-${year}`;
+
+    const date = {
+        day_name: dayName,
+        day_of_month: dayOfMonth,
+        month_number: monthNumber + 1,
+        month_name: monthName,
+        year: year
+    };
 
     return date;
 }
 
+function getMonthName(monthNumber) {
+    const monthNames = require('helpers/modules/time/config/months.json');
```

Comment:
Unless new months are added in the near future I don't think its necessary that we require the config json in each function call. Same for `getDayName()` - lets move those at the top file level so they're only ever called once.